MultimodalQnA/docker_compose/amd/gpu/rocm/README.md

```diff
@@ -28,10 +28,10 @@ cd GenAIComps
 docker build --no-cache -t opea/embedding-multimodal-bridgetower:latest --build-arg EMBEDDER_PORT=$EMBEDDER_PORT --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/embeddings/src/integrations/dependency/bridgetower/Dockerfile .
 ```
 
-Build embedding-multimodal microservice image
+Build embedding-tei microservice image
 
 ```bash
-docker build --no-cache -t opea/embedding-multimodal:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/embeddings/src/Dockerfile .
+docker build --no-cache -t opea/embedding-tei:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/embeddings/src/Dockerfile .
```

Comment:
opea/embedding:latest ?? https://github.com/opea-project/GenAIComps/pull/1073/files

Reply:
I will merge this PR first, need to fix the issue ASAP.